### ISSUE TITLE
Safety Harness uses "afflictiontypes" instead of "afflictionidentifiers" for lacerations and burns

### ISSUE THREAD

**Issue by Jade-Harleyy** [Regalis11/Barotrauma#9886]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The "Safety Harness" `id: safetyharness` uses the `afflictiontypes` tagfor burn and lacerations resistance instead of the `afflictionidentifiers` tag, which makes the resistances not work, which was revealed through testing.

### Reproduction steps

1. Start a game.
2. Spawn a Safety Harness and wear it.
3. Fail skill checks on broken devices.
4. Observe how damage is not reduced.

### Bug prevalence

Happens every time I play

### Version

0.18.15.1 / 0.18.15.2 on macOS

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
There doesn't seem to be any problem with the burn modifier: "burn" is an affliction type, so using it here is correct. I gave this a test too and the safety harness does seem to reduce burns.

Fixed the lacerations modifier in https://github.com/Regalis11/Barotrauma-development/commit/88cb418f9a9741a9d1ce7ddb54f0197450908713. While fixing this I also noticed the concussion modifiers on helmets had the same problem, and fixed it in the same commit.
